app/services/planned_disbursement_history.rb

```diff
@@ -24,6 +24,10 @@ def set_value(value)
     end
   end
 
+  def clear
```

Comment:
Since this is destructive, perhaps it should be called `clear!` ?

Reply:
Nice, done 👍